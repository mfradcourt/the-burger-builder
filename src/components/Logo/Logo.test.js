import React from 'react';
import { shallow } from "enzyme";

import Logo from './Logo';

describe('<Logo />', () => {
  let wrapper;

  it('should render Logo', () => {
    wrapper = shallow(
      <Logo/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
