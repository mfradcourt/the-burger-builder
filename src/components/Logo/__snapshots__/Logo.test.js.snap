// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render Logo 1`] = `
<div
  className="Logo"
>
  <img
    alt="MyBurger"
    src="burger-logo.png"
  />
</div>
`;
